src/integration-test/resources/ErrandCommunicationIT/__files/test01_sendEmail/mappings/api-messaging-send-email.json

```diff
@@ -12,12 +12,13 @@
 			"equalToJson" : {
 				"emailAddress": "john.a@zoidberg.com",
 				"subject": "Insane in the Mainframe",
-				"message": "Some message about the HAL Institute for Criminally Insane Robots, where robots that murderously turns against its human crew are held",
+				"htmlMessage": "<html>Some message about the HAL Institute for <b>Criminally Insane Robots</b>, where robots that <i>murderously</i> turns against its human crew are held</html>",
```

Comment:
Is it always HTML?
Does the frontEnd have an option to send messages in plain/text as well?

(you normally populates several parts when sending email. When sending HTML-mails, it's normal to also set a text/plain-part for clients that doesn't support HTML (or for people that have disabled HTML-mails)

Reply:
UI will only send html-formatted text to backend. I can check with Henrik if he will ever send a plain text.

Reply:
Talked to Henrik and we decided to update API with new attribute to provide both html and plain message. Service is updated accordingly but no new version will be created in WSO2 (version 2.0 will be updated and Henrik will adapt to the changes).